src/components/studies/app-design/IrbBoardContactSection.tsx

```diff
@@ -222,11 +231,17 @@ const IrbBoardContactSection: React.FunctionComponent<IrbBoardContactSectionProp
             titleText="Email*"
           />
           {!emailErrorState.isIrbEmailValid && (
-            <FormHelperText
-              id="ethics-bad-email-text"
-              className={classes.errorText}>
-              email should be in a valid format such as: example@placeholder.com
-            </FormHelperText>
+            <AlertWithText
```

Comment:
can we have the icon be a part of AlertWithText?
If we have several possibilities we can have a variant that would add a class/image combo.
If it is challenging with StyledComponent (you might be able to use 'content' and :before in css to get it working) you can replace it w/  a regular component that uses styled component